src/nile/core/node.py

```diff
@@ -6,7 +6,7 @@
 from nile.common import DEFAULT_GATEWAYS, write_node_json
 
 
-def node(host="127.0.0.1", port=5050, seed=None, lite_mode=False):
+def node(host="127.0.0.1", port=5050, node_args=None):
```

Comment:
I just wanted to let the defaults in our control, but if they match I agree it makes sense to remove them.

Reply:
On second thought. I will let the host and port options as a commodity if you agree because we are using them here:

```
        if host == "127.0.0.1":
            network = "localhost"
        else:
            network = host
        gateway_url = f"http://{host}:{port}/"
        if DEFAULT_GATEWAYS.get(network) != gateway_url:
            write_node_json(network, gateway_url)
```

And if we removed them, I would need to parse the node_args to get the values anyway (is easier to catch them separately as we are currently doing).

Reply:
Ahh right! Yeah, leaving them in sounds good to me. Depending on how the Nile config turns out down the road, maybe we can revisit this.

Reply:
Sounds good.